torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -299,7 +299,7 @@ RegisterOperators reg({
 // e.g. s - d == -d + s
 
 #define DEFINE_ST_OP(aten_op, reverse_exp)                             \
-  Operator("aten::" #aten_op "(Scalar a, Tensor b) -> Tensor", [](Node* node) { \
+  Operator("aten::" #aten_op "(Scalar other, Tensor self) -> Tensor", [](Node* node) { \
```

Comment:
This was a necessary change because `attr::a` and `attr::b` don't exist (register_prim_ops doesn't inject symbols into aten_interned_strings). I think this rename makes more sense because one can do `torch.add(other=1, self=tensor)` in python land.